src/db/models.py

```diff
@@ -1,24 +1,64 @@
 from . import connection
+import psycopg2.sql as sql
 
 class ApplicationsModel:
+    TableName = "applications"
+    Model = {}
+
+    @classmethod
+    def setModel(cls, fieldnames, sample_data):
```

Comment:
Make this an instance method

Reply:
schema dependent, I don't want to allow any instance to have an invalid schema should anyone try to update the database